docs/getting-started/elements.md

```diff
@@ -0,0 +1,34 @@
+## Installation
+
+The Warp Elements package can be installed from NPM.
+
+### with npm: 
+```shell
+npm install @warp-ds/elements
+```
+
+### with pnpm
+```shell
+pnpm add @warp-ds/elements
+```
+
+## Using Components
+
+Once installed, components can be used in your HTML markup.
+
+```js
+<f-breadcrumbs class="mt-10">
+  <a href="#/url/1">Eiendom</a>
+  <a href="#/url/2">Bolig til salgs</a>
+  <a href="#/url/3" aria-current="page"> Oslo </a>
+</f-breadcrumbs>
```

Comment:
Should this be w-breadcrumbs now? 🙂 
...personally I would prefer Norwegian as now, but do we want the examples to be in English? 😄

Reply:
...and this code block should be html as well. 😄

Reply:
Good point! Changed to English examples in 01bf8aab15752d40d7fce574788aaf33668ec29d